Source/Csla/Serialization/Mobile/CustomSerializers/ClaimsPrincipalSerializer.cs

```diff
@@ -0,0 +1,120 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ClaimsPrincipalFormatter.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>A formatter that can serialize and deserialize a ClaimsPrincipal object</summary>
+//-----------------------------------------------------------------------
+
+using System.Security.Claims;
+
+namespace Csla.Serialization.Mobile.CustomSerializers;
+
+#if NET8_0_OR_GREATER || NETSTANDARD2_0_OR_GREATER
+/// <summary>
+/// A formatter that can serialize and deserialize a ClaimsPrincipal object.
+/// </summary>
+public class ClaimsPrincipalSerializer : IMobileSerializer
```

Comment:
Any good reason to make them `public` instead of `internal`?
`internal` would save us from beeing unable to change them because that would be a breaking change. Furthermore if someone _really_ wants that serializer they have two options.
Ask us and we could provide a new method which retrieves the serializer for a `ClaimsPrincipal` or (my preferred variant) the user can just retrieve _all_ known serializers and use the one who says "I can process `ClaimsPrincipal`".

Reply:
Good point, I'll make this internal